src/test/java/mate/academy/onlinebookstoreproject/controller/BookControllerTest.java

```diff
@@ -0,0 +1,228 @@
+        theGreatGatsbyBookDto.setIsbn("978-0-13-235088-4");
+
+        vanityFairBookDto = new BookDto();
+        vanityFairBookDto.setId(2L);
+        vanityFairBookDto.setTitle("Vanity Fair");
+        vanityFairBookDto.setAuthor("William Thackeray");
+        vanityFairBookDto.setPrice(new BigDecimal("90.00"));
+        vanityFairBookDto.setDescription("A classic novel of social satire");
+        vanityFairBookDto.setCategoryIds(Set.of(2L));
+        vanityFairBookDto.setCoverImage("vanity-fair.jpg");
+        vanityFairBookDto.setIsbn("978-0-19-953762-4");
+
+        updateBookRequestDto = new CreateBookRequestDto();
+        updateBookRequestDto.setTitle("The Great Gatsby UPDATED");
+        updateBookRequestDto.setAuthor("F. Scott Fitzgerald UPDATED");
+        updateBookRequestDto.setPrice(new BigDecimal("100.00"));
+        updateBookRequestDto.setDescription("A novel about american dream UPDATED");
+        updateBookRequestDto.setCoverImage("the-great-gatsby.jpg");
+        updateBookRequestDto.setCategoryIds(Set.of(1L));
+        updateBookRequestDto.setIsbn("978-0-13-235088-4");
+
+        updateBookResponseDto = new BookDto();
+        updateBookResponseDto.setId(1L);
+        updateBookResponseDto.setTitle("The Great Gatsby UPDATED");
+        updateBookResponseDto.setAuthor("F. Scott Fitzgerald UPDATED");
+        updateBookResponseDto.setPrice(new BigDecimal("100.00"));
+        updateBookResponseDto.setDescription("A novel about american dream UPDATED");
+        updateBookResponseDto.setCoverImage("the-great-gatsby.jpg");
+        updateBookResponseDto.setCategoryIds(Set.of(1L));
+        updateBookResponseDto.setIsbn("978-0-13-235088-4");
+
+        flowersForAlgernonCreateDto = new CreateBookRequestDto();
+        flowersForAlgernonCreateDto.setTitle("Flowers for Algernon");
+        flowersForAlgernonCreateDto.setAuthor("Daniel Keyes");
+        flowersForAlgernonCreateDto.setPrice(new BigDecimal("15.99"));
+        flowersForAlgernonCreateDto.setDescription("A science fiction novel");
+        flowersForAlgernonCreateDto.setCoverImage("flowers-for-algernon.jpg");
+        flowersForAlgernonCreateDto.setCategoryIds(Set.of(1L));
+        flowersForAlgernonCreateDto.setIsbn("978-3-1136-5552-7");
+
+        flowersForAlgernonDto = new BookDto();
+        flowersForAlgernonDto.setId(1L);
+        flowersForAlgernonDto.setTitle("Flowers for Algernon");
+        flowersForAlgernonDto.setAuthor("Daniel Keyes");
+        flowersForAlgernonDto.setPrice(new BigDecimal("15.99"));
+        flowersForAlgernonDto.setDescription("A science fiction novel");
+        flowersForAlgernonDto.setCoverImage("flowers-for-algernon.jpg");
+        flowersForAlgernonDto.setCategoryIds(Set.of(1L));
+        flowersForAlgernonDto.setIsbn("978-3-1136-5552-7");
+    }
+
+    @WithMockUser(username = "user")
+    @Test
+    @DisplayName("Get a list of 2 books mapped to dto from the DB ")
+    void getAll_Return_TwoBookDto() throws Exception {

```

Comment:
This name is not fully correct. Please check some guidance. ([like this](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea)) It should have a pattern:  MethodName_StateUnderTest_ExpectedBehavior